azurerm/resource_arm_key_vault_test.go

```diff
@@ -102,6 +102,8 @@ func TestAccAzureRMKeyVault_complete(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testCheckAzureRMKeyVaultExists(resourceName),
 					resource.TestCheckResourceAttrSet(resourceName, "access_policy.0.application_id"),
+					resource.TestCheckResourceAttr(resourceName, "enabled_for_soft_delete", "true"),
+					resource.TestCheckResourceAttr(resourceName, "enabled_for_purge_protection", "true"),
```

Comment:
rather than modifying this test - can we check these values in a new test?

Reply:
I created a seperate test for this: `TestAccAzureRMKeyVault_enable_soft_delete`